my-webxr-app/src/components/DataPointMenu.tsx

```diff
@@ -14,12 +14,20 @@ export default function DataPointMenu(
 
   return (
     <Billboard visible={selectedDataPoint != null} {...billboardProps}>
-      <Plane args={[1.75, 0.25]}>
-        <Text fontSize={0.1} color={"black"}>
-          Here are data point #
-          {selectedDataPoint == null ? "-" : selectedDataPoint} properties!
-        </Text>
-      </Plane>
+        <Plane args={[1.25, 0.75]}>
+
+                <Text fontSize={0.075} color={"black"}>
+                    Here are data point # {selectedDataPoint == null ? "-" : selectedDataPoint.id} properties!
+                    {"\n\n"}
+                    Marker: {selectedDataPoint == null ? "-" : selectedDataPoint.marker + "\n"}
+                    Color: {selectedDataPoint == null ? "-" : selectedDataPoint.color + "\n"}
+                    x, y, z: {selectedDataPoint == null ? "-" : selectedDataPoint.meshProps?.position + "\n"}
+                    Column 1: {selectedDataPoint == null ? "-" : selectedDataPoint.column1 + "\n"}
```

Comment:
Might be nice to show let the title be:
Column X:
Column Y:
....
instead of Column 1, 2

Reply:
If the user selects 4 columns, would we start back at A? Or is it always guaranteed to be 3?